README.md

```diff
@@ -1 +1,84 @@
-# leaderboard-project
\ No newline at end of file
+# leaderboard-project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+In this project, I set up my To-Do List web page. My goal was add the functionality to add and display each todo task inside my task list. 
```

Comment:
Kindly update your description and make it relevant to this project.

Reply:
ok ok thank your so much I forgot to update.